README.md

```diff
@@ -92,6 +128,61 @@ Install the requirements mentioned above and copy the distribution into
 any directory you like.
 
 
+Execute
+-------
+
+```
+python2 ../ci-dnf-stack/cidnfstack.py build -c dnf-pull-requests tito
+```
+Localtest (omly for project that use tito):
+```
+python2 /path/to/cidnfstack.py build
+```
+Will test the project in current directory.
+
+Or:
+```
+python2 /path/to/cidnfstack.py build -l /path/to/project/1/ -l /path/to/project/2/
```

Comment:
Yes, there is option setup that is used for setup Copr repository. But I didn't code it and never used.